Sample/Warehouse/Warehouse.Api.Tests/Products/RegisteringProduct/RegisterProductTests.cs

```diff
@@ -62,16 +62,16 @@ await API.Given()
     private static string ValidSKU => $"CC{DateTime.Now.Ticks}";
     private const string ValidDescription = "VALID_DESCRIPTION";
 
-    public static TheoryData<RegisterProductRequest> ValidRequests = new()
-    {
+    public static TheoryData<RegisterProductRequest> ValidRequests =
```

Comment:
@SimonCropp, could you revert this part? Rider was too quick here. I once also tried to apply refactoring here, but it's a false positive, and XUnit is not dealing with it correctly.

Reply:
fixed

Reply:
Thank you 👌